lib/CapabilityDelegation.js

```diff
@@ -79,6 +80,8 @@ module.exports = class CapabilityDelegation extends ControllerProofPurpose {
       // see if the parent capability has already been verified
       let {verifiedParentCapability} = this;
       if(!verifiedParentCapability) {
+        console.log('FIXME_I_DONT_RUN');
```

Comment:
@dlongley nothing in our test suite causes this code path to be executed.  Do we need an additional test, or is there still some problem?

Reply:
@mattcollier -- maybe we don't have any tests that directly verify capability delegation (without also verifying invocation). We should add such tests if we don't have them.

Reply:
Yup, no `verify` calls in the test suite pass in a `CapabilityDelegation` purpose at the moment. We should duplicate some of the invocation tests and just check the delegation proofs without checking the invocation part. That should trigger that code to run.